Shared/Store/BaseAccountStore.swift

```diff
@@ -45,6 +48,11 @@ class BaseAccountStore {
             return
         }
 
+        if !networkHelper.isConnectedToNetwork {
+            self._syncCredentials.onNext(OfflineSyncCredential)
+            return
+        }
```

Comment:
Is this ever not connected and not logged in?

Reply:
no; if we don't have a stored `FirefoxAccount` we don't run this code.